### PR TITLE
Install and configure oozie-client on worker node

### PR DESCRIPTION
**Pull request by kiiranh**
This fixes issue #60

### CODE REVIEW

File: cookbooks/bcpc-hadoop/recipes/oozie_client.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Cookbook Name : bcpc-hadoop
+# Recipe Name : oozie_client
+# Description : To setup oozie-client
+
+include_recipe 'bcpc-hadoop::oozie_config'
+
+package "oozie-client" do
+   action :install
```
Comment (bijugs) [inline cookbooks/bcpc-hadoop/recipes/oozie_client.rb]: We may want to use ``upgrade`` action as a standard.

**Comment by kiiranh** [comment]
Made the change. I will squash the changes after getting final OK.

**Comment by bijugs** [comment]
LGTM.

**Comment by kiiranh** [comment]
Squashed commits.

**Comment by cbaenziger** [comment]
Indeed LGTM

**Pull request merged**